Packs/Malware/Playbooks/playbook-Malware_Investigation_-_Manual_README.md

```diff
@@ -1,3 +1,9 @@
+##Deprecated
```

Comment:
add 1 space here to make it a header

Reply:
![image](https://user-images.githubusercontent.com/88268646/171643544-a0e06548-ab38-4267-a3db-982e0188e186.png)
But I'll add it, to be on the safe side.